docs/src/reference/variables.md

```diff
@@ -9,12 +9,6 @@ DocTestFilters = [r"MathOptInterface|MOI"]
 
 # Variables
 
-## Types
-
-```@docs
-VariableIndex
```

Comment:
Why removing this ?

Reply:
it's also in another place, I would have kept here too probably

Reply:
It's in the Standard Form section. It's a toss-up where it should go. We have a problem that it's both an index and a function. We're confusing the abstraction.